tests/neg/tailcall/t1672b.scala

```diff
@@ -46,7 +46,7 @@ object Test1772B {
     else 1 + (try {
       throw new RuntimeException
     } catch {
-      case _: Throwable => bar(i - 1) // old-error
+      case _: Throwable => bar(i - 1) // error
```

Comment:
why was this test affected? It doesn't seem to have anything to do with by-name changes.